Documentation/ceph-monitoring.md

```diff
@@ -80,6 +80,15 @@ Below the `Execute` button, ensure the `Graph` tab is selected and you should no
 You can find Prometheus Consoles here: https://github.com/ceph/cephmetrics/tree/master/dashboards/current.
 A guide to how you can write your own Prometheus consoles can be found on the official Prometheus site here: https://prometheus.io/docs/visualization/consoles/.
 
+## Prometheus Alerts
+To enable prometheus based alerting for ceph, run the following commands:  
+```
+cd cluster/examples/kubernetes/ceph/monitoring
+kubectl create -f prometheus-ceph-rules.yaml
+```
+
+> **NOTE** Change `Namespace: rook-ceph` in `prometheus-ceph-rules.yaml` as needed.
```

Comment:
This note would also apply to all of the other yaml files in this `monitoring` folder as well and in all the rook examples the namespace `rook-ceph` is used. Could we remove this note since this is implied with all the rook examples?

Reply:
Made the note clearer. This works? or should I remove it?

Reply:
@umangachapagain I'd suggest to remove it, thanks

Reply:
ack.